### ISSUE TITLE
iPad:  No assignment after return from Letter

### ISSUE THREAD

**Issue by tonyamurray** [yeatmanlab/roar#48]
I logged in as 1036test001 (password: test001) and correctly saw both Letter and Phoneme.  I played Letter to the end.  When I returned to the dashboard, "You have no assignments!" was displayed.  I clicked Sign-out, then logged in again.  Letter was correctly checked and Phoneme was present.

This bug is Critical for FOC the week of 9/11 and all the Reading Reimagined schools who use dashboard login the week of 9/18.

**Comment by tonyamur** [comment]
I next played Phoneme.  On return to the dashboard, I again saw "You have no assignments!" where I should have seen both games completed.  Refreshing showed a checkmark for Letter but not for Phoneme.  

Signing out with the button in the upper right corner and then back in showed "You have no assignments!"

Signing out with the button in the lower middle of the screen and back in showed both games correctly checked complete.

**Comment by ctflores** [comment]
I also tested this and had the same experience. After completing Letter, it gave me this screen 
<img width="1347" alt="image" src="https://github.com/yeatmanlab/roar/assets/35785147/6c27b210-5cf9-4201-92a8-6f8ba01424cb">

**Comment by ctflores** [comment]
This is a huge problem because Kindergarteners have a hard enough time logging in. 
These students enter using Username and Password.

**Comment by richford** [comment]
So I checked this administration, and it wasn’t supposed to start until midnight tonight. I think the “you have no assignments” screen was actually the correct one to show and the fact that you were able to see the administration the first time around is the actual bug.

I just tested this by:
- editing the administration to start this morning
- testing with user 1036test001

It worked with this change.

This would also explain why we didn’t experience this behavior with any other orgs: they didn’t have administrations that hadn’t started yet.

So I still need to figure out why the initial check is failing and students are able to see administrations that have been created but aren’t supposed to be “open” yet. But I think for tomorrow, since the administration will be open, that this bug will not occur.
